src/openhim.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
```

Comment:
I think we can remove this blank line and group it as part of the included npm modules